src/components/ValidateSubscriber/ValidateMaxSubscribersForm.tsx

```diff
@@ -71,7 +90,7 @@ export const ValidateMaxSubscribersForm = ({
     }
 
     if (isCheckbox(questionItem.answer)) {
-      const fieldName = `answer${questionIndex}`;
+      let fieldName = `answer${questionIndex}`;
```

Comment:
Just curiosity, why did you make it mutable?

Reply:
It's a error, I will rollback